src/modals/UnlockChunks/Overview.tsx

```diff
@@ -25,6 +28,14 @@ export const Overview = forwardRef(
     const { isFastUnstaking } = useUnstaking();
     const { t } = useTranslation('modals');
 
+    const { timeleft: eraTimeLeft } = useEraTimeLeft();
+    const { fromNow, timeleftAsString } = useTimeLeft({
+      refreshInterval: 60,
+      refreshCallback: () => eraTimeLeft,
```

Comment:
Bit of a detour but can these be optional? Default `refreshInterval` to 60 and only call `refreshCallback` if it is provided.